### PR TITLE
Store S2 cells

### PR DESCRIPTION
**Pull request by Fabio1988**
@unseenmagik thanks for initial contribution :) 

Store GMO map cells

Update `updated` timestamp of pokestop, gym, s2cell regularly in a certain time windows (every 15 minutes)

Clear Old Forts after 30 min unseen

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -413,7 +416,14 @@ func decodeGMO(ctx context.Context, sDec []byte, username string) string {
 			if fort.ActivePokemon != nil {
 				newMapPokemon = append(newMapPokemon, decoder.RawMapPokemonData{Cell: mapCell.S2CellId, Data: fort.ActivePokemon})
 			}
+			// collect fort information of cell
+			if fort.FortType == pogo.FortType_CHECKPOINT {
```
Comment (Fabio1988) [inline main.go:420]: @jfberry correct me if that is the wrong approach in Golang to compare ENUM Literals :)
is there an exhaustive way to compare?

In Kotlin /Java you could do something like that:
```kotlin
when (fort.FortType) {
  pogo.FortType_CHECKPOINT -> ....
  pogo.FortType_GYM -> ...
}
```

**Comment by Fabio1988** [comment]
I would appreciate your review guys :) not tested yet, need to setup a non-production environment first :+1:

File: main.go (hunk 1 above)
Comment (jfberry) [reply main.go:420]: switch statement is what you are looking for
https://gobyexample.com/switch

File: main.go [hunk 2]
```diff
@@ -413,7 +416,15 @@ func decodeGMO(ctx context.Context, sDec []byte, username string) string {
 			if fort.ActivePokemon != nil {
 				newMapPokemon = append(newMapPokemon, decoder.RawMapPokemonData{Cell: mapCell.S2CellId, Data: fort.ActivePokemon})
 			}
+			// collect fort information of cell
+			switch fort.FortType {
+			case pogo.FortType_CHECKPOINT:
+				stopIdsPerCell[mapCell.S2CellId] = append(stopIdsPerCell[mapCell.S2CellId], fort.FortId)
```
Comment (Fabio1988) [inline main.go:422]: is it possible to collect that information in that way @jfberry ?

```go
var stopIdsPerCell = make(map[uint64][]string)
```
appending the map value with content, or is it needed to initialize the slices? :)

**Pull request merged**